assets/src/application/ui/layout/espressoDropdownMenu/types.ts

```diff
@@ -0,0 +1,17 @@
+import { AnyObject } from '@appServices/utilities/types';
+
+export interface EspressoDropdownMenuProps extends AnyObject {
+	children: React.ReactNode;
+	className?: string;
+	icon?: string | null;
```

Comment:
no need to use `null` as union type when we use TS `strict: false`

Reply:
I've added that from here: https://github.com/WordPress/gutenberg/tree/master/packages/components/src/dropdown-menu#icon
***
but removed now, and left only:
`icon?: string;`